api/src/test/java/uk/gov/dhsc/htbhf/claimant/eligibility/EligibilityDecisionHandlerTest.java

```diff
@@ -318,9 +334,9 @@ void shouldExpirePendingExpiryClaim() {
 
     private void verifyClaimSavedWithStatus(ClaimStatus claimStatus, CardStatus cardStatus) {
         ArgumentCaptor<Claim> argumentCaptor = ArgumentCaptor.forClass(Claim.class);
-        verify(claimRepository, times(2)).save(argumentCaptor.capture());
```

Comment:
claimRepository.save() is only called once now with an updated claim and card status.